resources/views/profile.blade.php

```diff
@@ -0,0 +1,95 @@
+@extends('layouts.app')
+@section('content')
+    <div class="wrapper">
+        <div class="sidebar" data-color="green" >
+            @include('includes.sidebar')
+        </div>
+
+        <div class="main-panel">
+            <div class="container-fluid">
+                <div class="row">
+                    <div class="col-md-9">
+                        <h5>User Profile</h5>
+                        <hr>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col-md-3 text-center">
+                        <img src="http://split.app/images/default-profile-picture.jpg" id="profileImage" style="width: 200px;">
```

Comment:
2 comments:

1.) the src path for default should be called by using the asset() method. please change

2.) a condition statement should be included here that checks if a user has an uploaded image. if he/she have an image, render that. otherwise, render the default image